src/main/java/edu/kis/powp/jobs2d/command/visitor/CommandDeepCopyVisitor.java

```diff
@@ -0,0 +1,34 @@
+package edu.kis.powp.jobs2d.command.visitor;
+
+import edu.kis.powp.jobs2d.command.*;
+
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.List;
+
+public class CommandDeepCopyVisitor implements Visitor {
+
+    private final List<DriverCommand> driverCommands = new LinkedList<>();
+
+    @Override
+    public void visitOperateToCommand(OperateToCommand operateToCommand) {
+        driverCommands.add(new OperateToCommand(operateToCommand.getPoint().x, operateToCommand.getPoint().y));
```

Comment:
This approach makes the visitor not reusable and tricky to use (a client may be suprised that the current accept call contains a previously copied command?). 
Each visit method call should replace the contents of driverCommands variable (not augment it).